github.com/cockroachdb/cockroach/pkg/ccl/sqlccl: TestBackupAsOfSystemTime failed under stress

SHA: https://github.com/cockroachdb/cockroach/commits/40d1c60a1454453422946d63330fa0aec43d46a8

Parameters:
```
COCKROACH_PROPOSER_EVALUATED_KV=false
TAGS=
GOFLAGS=
```

Stress build found a failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=149212&tab=buildLog

```
W170214 07:57:30.966748 25000 server/status/runtime.go:116  Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I170214 07:57:30.967616 25000 server/config.go:482  1 storage engine initialized
I170214 07:57:30.967845 25000 server/node.go:442  [n?] store [n0,s0] not bootstrapped
I170214 07:57:30.969447 25000 server/node.go:371  [n?] **** cluster 1019cee0-c1f1-46c3-9106-42cb03bb9b06 has been created
I170214 07:57:30.969463 25000 server/node.go:372  [n?] **** add additional nodes by specifying --join=127.0.0.1:39125
I170214 07:57:30.969896 25000 storage/store.go:1261  [n1] [n1,s1]: failed initial metrics computation: [n1,s1]: system config not yet available
I170214 07:57:30.969919 25000 server/node.go:455  [n1] initialized store [n1,s1]: {Capacity:536870912 Available:536870912 RangeCount:1 LeaseCount:0}
I170214 07:57:30.969934 25000 server/node.go:340  [n1] node ID 1 initialized
I170214 07:57:30.969967 25000 gossip/gossip.go:293  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:39125" > attrs:<> locality:<> 
I170214 07:57:30.970029 25000 storage/stores.go:296  [n1] read 0 node addresses from persistent storage
I170214 07:57:30.970067 25000 server/node.go:587  [n1] connecting to gossip network to verify cluster ID...
I170214 07:57:30.970081 25000 server/node.go:611  [n1] node connected via gossip and verified as part of cluster "1019cee0-c1f1-46c3-9106-42cb03bb9b06"
I170214 07:57:30.970097 25000 server/node.go:390  [n1] node=1: started with [[]=] engine(s) and attributes []
I170214 07:57:30.970113 25000 sql/executor.go:333  [n1] creating distSQLPlanner with address {tcp 127.0.0.1:39125}
I170214 07:57:30.970880 24974 storage/split_queue.go:98  [split,n1,s1,r1/1:/M{in-ax},@c423269c00] splitting at key /Table/0/0
I170214 07:57:30.972394 25000 server/server.go:632  [n1] starting https server at 127.0.0.1:54768
I170214 07:57:30.972409 25000 server/server.go:633  [n1] starting grpc/postgres server at 127.0.0.1:39125
I170214 07:57:30.972416 25000 server/server.go:634  [n1] advertising CockroachDB node at 127.0.0.1:39125
I170214 07:57:30.975956 24974 storage/replica_command.go:2397  [split,n1,s1,r1/1:/M{in-ax},@c423269c00] initiating a split of this range at key /Table/0 [r2]
I170214 07:57:30.981472 24974 storage/split_queue.go:98  [split,n1,s1,r2/1:/{Table/0-Max},@c422dac380] splitting at key /Table/11/0
I170214 07:57:30.981897 24974 storage/replica_command.go:2397  [split,n1,s1,r2/1:/{Table/0-Max},@c422dac380] initiating a split of this range at key /Table/11 [r3]
I170214 07:57:30.983097 25000 sql/event_log.go:95  [n1] Event: "alter_table", target: 12, info: {TableName:eventlog Statement:ALTER TABLE system.eventlog ALTER COLUMN uniqueID SET DEFAULT uuid_v4() User:node MutationID:0 CascadeDroppedViews:[]}
I170214 07:57:30.988864 24974 storage/split_queue.go:98  [split,n1,s1,r3/1:/{Table/11-Max},@c423221500] splitting at key /Table/12/0
I170214 07:57:30.988897 24974 storage/replica_command.go:2397  [split,n1,s1,r3/1:/{Table/11-Max},@c423221500] initiating a split of this range at key /Table/12 [r4]
I170214 07:57:30.991452 25000 server/server.go:689  [n1] done ensuring all necessary migrations have run
I170214 07:57:30.991472 25000 server/server.go:691  [n1] serving sql connections
I170214 07:57:31.005358 24974 storage/split_queue.go:98  [split,n1,s1,r4/1:/{Table/12-Max},@c422248e00] splitting at key /Table/13/0
I170214 07:57:31.005393 24974 storage/replica_command.go:2397  [split,n1,s1,r4/1:/{Table/12-Max},@c422248e00] initiating a split of this range at key /Table/13 [r5]
I170214 07:57:31.012382 24974 storage/split_queue.go:98  [split,n1,s1,r5/1:/{Table/13-Max},@c423177880] splitting at key /Table/14/0
I170214 07:57:31.012419 24974 storage/replica_command.go:2397  [split,n1,s1,r5/1:/{Table/13-Max},@c423177880] initiating a split of this range at key /Table/14 [r6]
I170214 07:57:31.024817 25005 sql/event_log.go:95  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:39125} Attrs: Locality:} ClusterID:1019cee0-c1f1-46c3-9106-42cb03bb9b06 StartedAt:1487059050970084174}
W170214 07:57:31.058753 25000 gossip/gossip.go:1178  [n?] no incoming or outgoing connections
W170214 07:57:31.059378 25000 server/status/runtime.go:116  Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I170214 07:57:31.060741 25000 server/config.go:482  1 storage engine initialized
I170214 07:57:31.060960 25000 server/node.go:442  [n?] store [n0,s0] not bootstrapped
I170214 07:57:31.060978 25000 storage/stores.go:296  [n?] read 0 node addresses from persistent storage
I170214 07:57:31.061001 25000 server/node.go:587  [n?] connecting to gossip network to verify cluster ID...
I170214 07:57:31.082260 25257 gossip/client.go:131  [n?] started gossip client to 127.0.0.1:39125
I170214 07:57:31.082580 25087 gossip/server.go:214  [n1] received initial cluster-verification connection from {tcp 127.0.0.1:38562}
I170214 07:57:31.082622 25000 server/node.go:611  [n?] node connected via gossip and verified as part of cluster "1019cee0-c1f1-46c3-9106-42cb03bb9b06"
I170214 07:57:31.082772 25287 storage/stores.go:312  [n?] wrote 1 node addresses to persistent storage
I170214 07:57:31.087051 25000 kv/dist_sender.go:367  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I170214 07:57:31.087552 25000 server/node.go:333  [n?] new node allocated ID 2
I170214 07:57:31.087590 25000 gossip/gossip.go:293  [n2] NodeDescriptor set to node_id:2 address:<network_field:"tcp" address_field:"127.0.0.1:38562" > attrs:<> locality:<> 
I170214 07:57:31.087640 25000 server/node.go:390  [n2] node=2: started with [[]=] engine(s) and attributes []
I170214 07:57:31.087659 25000 sql/executor.go:333  [n2] creating distSQLPlanner with address {tcp 127.0.0.1:38562}
I170214 07:57:31.089987 25000 server/server.go:632  [n2] starting https server at 127.0.0.1:42530
I170214 07:57:31.090005 25000 server/server.go:633  [n2] starting grpc/postgres server at 127.0.0.1:38562
I170214 07:57:31.090013 25000 server/server.go:634  [n2] advertising CockroachDB node at 127.0.0.1:38562
I170214 07:57:31.090086 25214 server/node.go:568  [n2] bootstrapped store [n2,s2]
I170214 07:57:31.090434 25379 storage/stores.go:312  [n1] wrote 1 node addresses to persistent storage
I170214 07:57:31.090814 25000 server/server.go:689  [n2] done ensuring all necessary migrations have run
I170214 07:57:31.090832 25000 server/server.go:691  [n2] serving sql connections
I170214 07:57:31.091565 25139 storage/replica_raftstorage.go:414  [replicate,n1,s1,r5/1:/Table/1{3-4},@c423177880] generated preemptive snapshot 939e7fc6 at index 16
I170214 07:57:31.095832 25217 sql/event_log.go:95  [n2] Event: "node_join", target: 2, info: {Descriptor:{NodeID:2 Address:{NetworkField:tcp AddressField:127.0.0.1:38562} Attrs: Locality:} ClusterID:1019cee0-c1f1-46c3-9106-42cb03bb9b06 StartedAt:1487059051087626677}
W170214 07:57:31.100375 25000 gossip/gossip.go:1178  [n?] no incoming or outgoing connections
W170214 07:57:31.121569 25000 server/status/runtime.go:116  Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I170214 07:57:31.122450 25000 server/config.go:482  1 storage engine initialized
I170214 07:57:31.122678 25000 server/node.go:442  [n?] store [n0,s0] not bootstrapped
I170214 07:57:31.122697 25000 storage/stores.go:296  [n?] read 0 node addresses from persistent storage
I170214 07:57:31.122732 25000 server/node.go:587  [n?] connecting to gossip network to verify cluster ID...
I170214 07:57:31.124636 25139 storage/store.go:3278  [replicate,n1,s1,r5/1:/Table/1{3-4},@c423177880] streamed snapshot: kv pairs: 40, log entries: 6, 0ms
I170214 07:57:31.125028 25449 storage/replica_raftstorage.go:596  [n2,s2,r5/?:{-},@c421ff6a80] applying preemptive snapshot at index 16 (id=939e7fc6, encoded size=5579, 1 rocksdb batches, 6 log entries)
I170214 07:57:31.125249 25449 storage/replica_raftstorage.go:604  [n2,s2,r5/?:/Table/1{3-4},@c421ff6a80] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170214 07:57:31.125743 25139 storage/replica_command.go:3253  [replicate,n1,s1,r5/1:/Table/1{3-4},@c423177880] change replicas (remove {2 2 2}): read existing descriptor range_id:5 start_key:"\225" end_key:"\226" replicas:<node_id:1 store_id:1 replica_id:1 > next_replica_id:2 
I170214 07:57:31.128038 25461 storage/replica.go:2487  [n1,s1,r5/1:/Table/1{3-4},@c423177880] proposing ADD_REPLICA {NodeID:2 StoreID:2 ReplicaID:2}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2}]
I170214 07:57:31.128596 25139 storage/replica_raftstorage.go:414  [replicate,n1,s1,r6/1:/{Table/14-Max},@c4221a9880] generated preemptive snapshot 983248b4 at index 11
I170214 07:57:31.129227 25139 storage/store.go:3278  [replicate,n1,s1,r6/1:/{Table/14-Max},@c4221a9880] streamed snapshot: kv pairs: 9, log entries: 1, 1ms
I170214 07:57:31.129606 25469 storage/raft_transport.go:437  [n2] raft transport stream to node 1 established
I170214 07:57:31.129767 25340 storage/replica_raftstorage.go:596  [n2,s2,r6/?:{-},@c420e68e00] applying preemptive snapshot at index 11 (id=983248b4, encoded size=504, 1 rocksdb batches, 1 log entries)
I170214 07:57:31.129914 25340 storage/replica_raftstorage.go:604  [n2,s2,r6/?:/{Table/14-Max},@c420e68e00] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170214 07:57:31.130519 25139 storage/replica_command.go:3253  [replicate,n1,s1,r6/1:/{Table/14-Max},@c4221a9880] change replicas (remove {2 2 2}): read existing descriptor range_id:6 start_key:"\226" end_key:"\377\377" replicas:<node_id:1 store_id:1 replica_id:1 > next_replica_id:2 
I170214 07:57:31.133267 25491 storage/replica.go:2487  [n1,s1,r6/1:/{Table/14-Max},@c4221a9880] proposing ADD_REPLICA {NodeID:2 StoreID:2 ReplicaID:2}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2}]
I170214 07:57:31.133801 25139 storage/replica_raftstorage.go:414  [replicate,n1,s1,r3/1:/Table/1{1-2},@c423221500] generated preemptive snapshot 0c1bad30 at index 22
I170214 07:57:31.134199 25139 storage/store.go:3278  [replicate,n1,s1,r3/1:/Table/1{1-2},@c423221500] streamed snapshot: kv pairs: 10, log entries: 12, 0ms
I170214 07:57:31.134569 25483 storage/replica_raftstorage.go:596  [n2,s2,r3/?:{-},@c423a23500] applying preemptive snapshot at index 22 (id=0c1bad30, encoded size=6057, 1 rocksdb batches, 12 log entries)
I170214 07:57:31.134824 25483 storage/replica_raftstorage.go:604  [n2,s2,r3/?:/Table/1{1-2},@c423a23500] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170214 07:57:31.135365 25139 storage/replica_command.go:3253  [replicate,n1,s1,r3/1:/Table/1{1-2},@c423221500] change replicas (remove {2 2 2}): read existing descriptor range_id:3 start_key:"\223" end_key:"\224" replicas:<node_id:1 store_id:1 replica_id:1 > next_replica_id:2 
I170214 07:57:31.142385 25522 storage/replica.go:2487  [n1,s1,r3/1:/Table/1{1-2},@c423221500] proposing ADD_REPLICA {NodeID:2 StoreID:2 ReplicaID:2}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2}]
I170214 07:57:31.142991 25139 storage/replica_raftstorage.go:414  [replicate,n1,s1,r2/1:/Table/{0-11},@c422dac380] generated preemptive snapshot 1b879a2a at index 24
I170214 07:57:31.152439 25139 storage/store.go:3278  [replicate,n1,s1,r2/1:/Table/{0-11},@c422dac380] streamed snapshot: kv pairs: 33, log entries: 14, 9ms
I170214 07:57:31.153015 25526 storage/replica_raftstorage.go:596  [n2,s2,r2/?:{-},@c422135500] applying preemptive snapshot at index 24 (id=1b879a2a, encoded size=12538, 1 rocksdb batches, 14 log entries)
I170214 07:57:31.153244 25526 storage/replica_raftstorage.go:604  [n2,s2,r2/?:/Table/{0-11},@c422135500] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170214 07:57:31.153625 25139 storage/replica_command.go:3253  [replicate,n1,s1,r2/1:/Table/{0-11},@c422dac380] change replicas (remove {2 2 2}): read existing descriptor range_id:2 start_key:"\210" end_key:"\223" replicas:<node_id:1 store_id:1 replica_id:1 > next_replica_id:2 
I170214 07:57:31.165956 25310 storage/replica.go:2487  [n1,s1,r2/1:/Table/{0-11},@c422dac380] proposing ADD_REPLICA {NodeID:2 StoreID:2 ReplicaID:2}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2}]
I170214 07:57:31.166781 25430 gossip/client.go:131  [n?] started gossip client to 127.0.0.1:39125
I170214 07:57:31.167217 25513 gossip/server.go:214  [n1] received initial cluster-verification connection from {tcp 127.0.0.1:52722}
I170214 07:57:31.167459 25000 server/node.go:611  [n?] node connected via gossip and verified as part of cluster "1019cee0-c1f1-46c3-9106-42cb03bb9b06"
I170214 07:57:31.167692 25514 storage/stores.go:312  [n?] wrote 1 node addresses to persistent storage
I170214 07:57:31.167736 25514 storage/stores.go:312  [n?] wrote 2 node addresses to persistent storage
I170214 07:57:31.169362 25139 storage/replica_raftstorage.go:414  [replicate,n1,s1,r1/1:/{Min-Table/0},@c423269c00] generated preemptive snapshot 26f39eed at index 52
I170214 07:57:31.173327 25533 storage/replica_raftstorage.go:596  [n2,s2,r1/?:{-},@c423487180] applying preemptive snapshot at index 52 (id=26f39eed, encoded size=179462, 1 rocksdb batches, 42 log entries)
I170214 07:57:31.174272 25533 storage/replica_raftstorage.go:604  [n2,s2,r1/?:/{Min-Table/0},@c423487180] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170214 07:57:31.175044 25139 storage/store.go:3278  [replicate,n1,s1,r1/1:/{Min-Table/0},@c423269c00] streamed snapshot: kv pairs: 579, log entries: 42, 5ms
I170214 07:57:31.178511 25139 storage/replica_command.go:3253  [replicate,n1,s1,r1/1:/{Min-Table/0},@c423269c00] change replicas (remove {2 2 2}): read existing descriptor range_id:1 start_key:"" end_key:"\210" replicas:<node_id:1 store_id:1 replica_id:1 > next_replica_id:2 
I170214 07:57:31.181142 25000 kv/dist_sender.go:367  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I170214 07:57:31.181497 25587 storage/replica.go:2487  [n1,s1,r1/1:/{Min-Table/0},@c423269c00] proposing ADD_REPLICA {NodeID:2 StoreID:2 ReplicaID:2}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2}]
I170214 07:57:31.181781 25000 server/node.go:333  [n?] new node allocated ID 3
I170214 07:57:31.181822 25000 gossip/gossip.go:293  [n3] NodeDescriptor set to node_id:3 address:<network_field:"tcp" address_field:"127.0.0.1:52722" > attrs:<> locality:<> 
I170214 07:57:31.181864 25000 server/node.go:390  [n3] node=3: started with [[]=] engine(s) and attributes []
I170214 07:57:31.181885 25000 sql/executor.go:333  [n3] creating distSQLPlanner with address {tcp 127.0.0.1:52722}
I170214 07:57:31.182059 25139 storage/replica_raftstorage.go:414  [replicate,n1,s1,r4/1:/Table/1{2-3},@c422248e00] generated preemptive snapshot b740de7f at index 21
I170214 07:57:31.183087 25139 storage/store.go:3278  [replicate,n1,s1,r4/1:/Table/1{2-3},@c422248e00] streamed snapshot: kv pairs: 25, log entries: 11, 1ms
I170214 07:57:31.183341 25545 storage/replica_raftstorage.go:596  [n2,s2,r4/?:{-},@c423262380] applying preemptive snapshot at index 21 (id=b740de7f, encoded size=7555, 1 rocksdb batches, 11 log entries)
I170214 07:57:31.183586 25000 server/server.go:632  [n3] starting https server at 127.0.0.1:36106
I170214 07:57:31.183602 25000 server/server.go:633  [n3] starting grpc/postgres server at 127.0.0.1:52722
I170214 07:57:31.183610 25000 server/server.go:634  [n3] advertising CockroachDB node at 127.0.0.1:52722
I170214 07:57:31.184851 25623 storage/stores.go:312  [n1] wrote 2 node addresses to persistent storage
I170214 07:57:31.185023 25624 storage/stores.go:312  [n2] wrote 2 node addresses to persistent storage
I170214 07:57:31.185622 25000 server/server.go:689  [n3] done ensuring all necessary migrations have run
I170214 07:57:31.185633 25000 server/server.go:691  [n3] serving sql connections
I170214 07:57:31.187670 25575 server/node.go:568  [n3] bootstrapped store [n3,s3]
I170214 07:57:31.188496 25578 sql/event_log.go:95  [n3] Event: "node_join", target: 3, info: {Descriptor:{NodeID:3 Address:{NetworkField:tcp AddressField:127.0.0.1:52722} Attrs: Locality:} ClusterID:1019cee0-c1f1-46c3-9106-42cb03bb9b06 StartedAt:1487059051181851979}
I170214 07:57:31.219540 25545 storage/replica_raftstorage.go:604  [n2,s2,r4/?:/Table/1{2-3},@c423262380] applied preemptive snapshot in 36ms [clear=0ms batch=0ms entries=0ms commit=32ms]
I170214 07:57:31.220179 25139 storage/replica_command.go:3253  [replicate,n1,s1,r4/1:/Table/1{2-3},@c422248e00] change replicas (remove {2 2 2}): read existing descriptor range_id:4 start_key:"\224" end_key:"\225" replicas:<node_id:1 store_id:1 replica_id:1 > next_replica_id:2 
I170214 07:57:31.224276 25724 storage/replica.go:2487  [n1,s1,r4/1:/Table/1{2-3},@c422248e00] proposing ADD_REPLICA {NodeID:2 StoreID:2 ReplicaID:2}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2}]
I170214 07:57:31.224661 25139 storage/queue.go:693  [n1,replicate] purgatory is now empty
I170214 07:57:31.224889 24975 storage/replica_raftstorage.go:414  [replicate,n1,s1,r5/1:/Table/1{3-4},@c423177880] generated preemptive snapshot becf11ad at index 29
I170214 07:57:31.242881 24975 storage/store.go:3278  [replicate,n1,s1,r5/1:/Table/1{3-4},@c423177880] streamed snapshot: kv pairs: 76, log entries: 19, 1ms
I170214 07:57:31.244220 25749 storage/replica_raftstorage.go:596  [n3,s3,r5/?:{-},@c422fabc00] applying preemptive snapshot at index 29 (id=becf11ad, encoded size=17176, 1 rocksdb batches, 19 log entries)
I170214 07:57:31.244510 25749 storage/replica_raftstorage.go:604  [n3,s3,r5/?:/Table/1{3-4},@c422fabc00] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170214 07:57:31.244954 24975 storage/replica_command.go:3253  [replicate,n1,s1,r5/1:/Table/1{3-4},@c423177880] change replicas (remove {3 3 3}): read existing descriptor range_id:5 start_key:"\225" end_key:"\226" replicas:<node_id:1 store_id:1 replica_id:1 > replicas:<node_id:2 store_id:2 replica_id:2 > next_replica_id:3 
I170214 07:57:31.248997 25742 storage/replica.go:2487  [n1,s1,r5/1:/Table/1{3-4},@c423177880] proposing ADD_REPLICA {NodeID:3 StoreID:3 ReplicaID:3}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2} {NodeID:3 StoreID:3 ReplicaID:3}]
I170214 07:57:31.251003 24975 storage/replica_raftstorage.go:414  [replicate,n1,s1,r1/1:/{Min-Table/0},@c423269c00] generated preemptive snapshot e404699a at index 68
I170214 07:57:31.260504 24975 storage/store.go:3278  [replicate,n1,s1,r1/1:/{Min-Table/0},@c423269c00] streamed snapshot: kv pairs: 790, log entries: 58, 9ms
I170214 07:57:31.268504 25779 storage/raft_transport.go:437  [n3] raft transport stream to node 1 established
I170214 07:57:31.268673 25762 storage/replica_raftstorage.go:596  [n3,s3,r1/?:{-},@c422134a80] applying preemptive snapshot at index 68 (id=e404699a, encoded size=256178, 1 rocksdb batches, 58 log entries)
I170214 07:57:31.269926 25762 storage/replica_raftstorage.go:604  [n3,s3,r1/?:/{Min-Table/0},@c422134a80] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=1ms]
I170214 07:57:31.270333 24975 storage/replica_command.go:3253  [replicate,n1,s1,r1/1:/{Min-Table/0},@c423269c00] change replicas (remove {3 3 3}): read existing descriptor range_id:1 start_key:"" end_key:"\210" replicas:<node_id:1 store_id:1 replica_id:1 > replicas:<node_id:2 store_id:2 replica_id:2 > next_replica_id:3 
I170214 07:57:31.279523 25417 storage/replica.go:2487  [n1,s1,r1/1:/{Min-Table/0},@c423269c00] proposing ADD_REPLICA {NodeID:3 StoreID:3 ReplicaID:3}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2} {NodeID:3 StoreID:3 ReplicaID:3}]
I170214 07:57:31.282019 24975 storage/replica_raftstorage.go:414  [replicate,n1,s1,r4/1:/Table/1{2-3},@c422248e00] generated preemptive snapshot f9bc8443 at index 26
I170214 07:57:31.282477 24975 storage/store.go:3278  [replicate,n1,s1,r4/1:/Table/1{2-3},@c422248e00] streamed snapshot: kv pairs: 31, log entries: 16, 0ms
I170214 07:57:31.282735 25568 storage/replica_raftstorage.go:596  [n3,s3,r4/?:{-},@c422dad880] applying preemptive snapshot at index 26 (id=f9bc8443, encoded size=10679, 1 rocksdb batches, 16 log entries)
I170214 07:57:31.282958 25568 storage/replica_raftstorage.go:604  [n3,s3,r4/?:/Table/1{2-3},@c422dad880] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170214 07:57:31.283415 24975 storage/replica_command.go:3253  [replicate,n1,s1,r4/1:/Table/1{2-3},@c422248e00] change replicas (remove {3 3 3}): read existing descriptor range_id:4 start_key:"\224" end_key:"\225" replicas:<node_id:1 store_id:1 replica_id:1 > replicas:<node_id:2 store_id:2 replica_id:2 > next_replica_id:3 
I170214 07:57:31.288747 25419 storage/replica.go:2487  [n1,s1,r4/1:/Table/1{2-3},@c422248e00] proposing ADD_REPLICA {NodeID:3 StoreID:3 ReplicaID:3}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2} {NodeID:3 StoreID:3 ReplicaID:3}]
I170214 07:57:31.292020 24975 storage/replica_raftstorage.go:414  [replicate,n1,s1,r3/1:/Table/1{1-2},@c423221500] generated preemptive snapshot b4549eb1 at index 25
I170214 07:57:31.292427 24975 storage/store.go:3278  [replicate,n1,s1,r3/1:/Table/1{1-2},@c423221500] streamed snapshot: kv pairs: 11, log entries: 15, 0ms
I170214 07:57:31.292652 25422 storage/replica_raftstorage.go:596  [n3,s3,r3/?:{-},@c421c0b180] applying preemptive snapshot at index 25 (id=b4549eb1, encoded size=7250, 1 rocksdb batches, 15 log entries)
I170214 07:57:31.292876 25422 storage/replica_raftstorage.go:604  [n3,s3,r3/?:/Table/1{1-2},@c421c0b180] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170214 07:57:31.293299 24975 storage/replica_command.go:3253  [replicate,n1,s1,r3/1:/Table/1{1-2},@c423221500] change replicas (remove {3 3 3}): read existing descriptor range_id:3 start_key:"\223" end_key:"\224" replicas:<node_id:1 store_id:1 replica_id:1 > replicas:<node_id:2 store_id:2 replica_id:2 > next_replica_id:3 
I170214 07:57:31.296284 25423 storage/replica.go:2487  [n1,s1,r3/1:/Table/1{1-2},@c423221500] proposing ADD_REPLICA {NodeID:3 StoreID:3 ReplicaID:3}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2} {NodeID:3 StoreID:3 ReplicaID:3}]
I170214 07:57:31.297438 24975 storage/replica_raftstorage.go:414  [replicate,n1,s1,r2/1:/Table/{0-11},@c422dac380] generated preemptive snapshot 96a5a568 at index 27
I170214 07:57:31.298195 24975 storage/store.go:3278  [replicate,n1,s1,r2/1:/Table/{0-11},@c422dac380] streamed snapshot: kv pairs: 34, log entries: 17, 1ms
I170214 07:57:31.298526 25802 storage/replica_raftstorage.go:596  [n3,s3,r2/?:{-},@c4202fb180] applying preemptive snapshot at index 27 (id=96a5a568, encoded size=13727, 1 rocksdb batches, 17 log entries)
I170214 07:57:31.298781 25802 storage/replica_raftstorage.go:604  [n3,s3,r2/?:/Table/{0-11},@c4202fb180] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170214 07:57:31.302075 24975 storage/replica_command.go:3253  [replicate,n1,s1,r2/1:/Table/{0-11},@c422dac380] change replicas (remove {3 3 3}): read existing descriptor range_id:2 start_key:"\210" end_key:"\223" replicas:<node_id:1 store_id:1 replica_id:1 > replicas:<node_id:2 store_id:2 replica_id:2 > next_replica_id:3 
I170214 07:57:31.305830 25784 storage/replica.go:2487  [n1,s1,r2/1:/Table/{0-11},@c422dac380] proposing ADD_REPLICA {NodeID:3 StoreID:3 ReplicaID:3}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2} {NodeID:3 StoreID:3 ReplicaID:3}]
I170214 07:57:31.306945 24975 storage/replica_raftstorage.go:414  [replicate,n1,s1,r6/1:/{Table/14-Max},@c4221a9880] generated preemptive snapshot cdea8512 at index 14
I170214 07:57:31.307450 24975 storage/store.go:3278  [replicate,n1,s1,r6/1:/{Table/14-Max},@c4221a9880] streamed snapshot: kv pairs: 10, log entries: 4, 0ms
I170214 07:57:31.307826 25847 storage/replica_raftstorage.go:596  [n3,s3,r6/?:{-},@c421951880] applying preemptive snapshot at index 14 (id=cdea8512, encoded size=1701, 1 rocksdb batches, 4 log entries)
I170214 07:57:31.308048 25847 storage/replica_raftstorage.go:604  [n3,s3,r6/?:/{Table/14-Max},@c421951880] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170214 07:57:31.308487 24975 storage/replica_command.go:3253  [replicate,n1,s1,r6/1:/{Table/14-Max},@c4221a9880] change replicas (remove {3 3 3}): read existing descriptor range_id:6 start_key:"\226" end_key:"\377\377" replicas:<node_id:1 store_id:1 replica_id:1 > replicas:<node_id:2 store_id:2 replica_id:2 > next_replica_id:3 
I170214 07:57:31.319047 25814 storage/replica.go:2487  [n1,s1,r6/1:/{Table/14-Max},@c4221a9880] proposing ADD_REPLICA {NodeID:3 StoreID:3 ReplicaID:3}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2} {NodeID:3 StoreID:3 ReplicaID:3}]
I170214 07:57:31.452034 25850 sql/event_log.go:95  [client=127.0.0.1:35932,user=root,n1] Event: "create_database", target: 50, info: {DatabaseName:bench Statement:CREATE DATABASE bench User:root}
I170214 07:57:31.452485 24974 storage/split_queue.go:98  [split,n1,s1,r6/1:/{Table/14-Max},@c4221a9880] splitting at key /Table/50/0
I170214 07:57:31.452520 24974 storage/replica_command.go:2397  [split,n1,s1,r6/1:/{Table/14-Max},@c4221a9880] initiating a split of this range at key /Table/50 [r7]
I170214 07:57:31.468318 25850 sql/event_log.go:95  [client=127.0.0.1:35932,user=root,n1] Event: "create_table", target: 51, info: {TableName:bench.bank Statement:CREATE TABLE bench.bank (id INT PRIMARY KEY, balance INT, payload STRING, FAMILY (id, balance, payload)) User:root}
I170214 07:57:31.472282 24974 storage/split_queue.go:98  [split,n1,s1,r7/1:/{Table/50-Max},@c420bfca80] splitting at key /Table/51/0
I170214 07:57:31.472317 24974 storage/replica_command.go:2397  [split,n1,s1,r7/1:/{Table/50-Max},@c420bfca80] initiating a split of this range at key /Table/51 [r8]
I170214 07:57:31.475683 25074 storage/replica_raftstorage.go:414  [raftsnapshot,n1,s1,r7/1:/{Table/50-Max},@c420bfca80] generated Raft snapshot 54fb4e56 at index 10
E170214 07:57:31.478248 25074 storage/queue.go:628  [raftsnapshot,n1,s1,r7/1:/{Table/50-Max},@c420bfca80] [n1,s1,r7/1:/{Table/50-Max}]: change replicas aborted due to failed preemptive snapshot: r7: remote couldn't accept snapshot with error: [n2,s2],r7: cannot apply snapshot: snapshot intersects existing range [n2,s2,r6/2:/{Table/14-Max}]
I170214 07:57:31.518918 25850 sql/mon/mem_usage.go:510  [client=127.0.0.1:35932,user=root,n1] txn: memory usage increases to 49 KiB (+49680)
I170214 07:57:31.518999 25850 sql/mon/mem_usage.go:510  [client=127.0.0.1:35932,user=root,n1] txn: memory usage increases to 64 KiB (+132)
I170214 07:57:31.519190 25850 sql/mon/mem_usage.go:510  [client=127.0.0.1:35932,user=root,n1] txn: memory usage increases to 128 KiB (+132)
I170214 07:57:31.559395 25944 storage/replica_command.go:2397  [n1,s1,r8/1:/{Table/51-Max},@c421416700] initiating a split of this range at key /Table/51/1/100 [r9]
I170214 07:57:31.566250 25074 storage/replica_raftstorage.go:414  [raftsnapshot,n1,s1,r8/1:/{Table/51-Max},@c421416700] generated Raft snapshot 49b57bdc at index 14
E170214 07:57:31.566715 25074 storage/queue.go:628  [raftsnapshot,n1,s1,r8/1:/{Table/51-Max},@c421416700] [n1,s1,r8/1:/{Table/51-Max}]: change replicas aborted due to failed preemptive snapshot: r8: remote couldn't accept snapshot with error: [n2,s2],r8: cannot apply snapshot: snapshot intersects existing range; initiated GC: [n2,s2,r7/2:/{Table/50-Max}]
I170214 07:57:31.590638 26004 storage/replica_command.go:2397  [n1,s1,r9/1:/{Table/51/1/1…-Max},@c4237d4380] initiating a split of this range at key /Table/51/1/200 [r10]
I170214 07:57:31.609792 25074 storage/replica_raftstorage.go:414  [raftsnapshot,n1,s1,r9/1:/{Table/51/1/1…-Max},@c4237d4380] generated Raft snapshot 35b6bce2 at index 11
E170214 07:57:31.610199 25074 storage/queue.go:628  [raftsnapshot,n1,s1,r9/1:/{Table/51/1/1…-Max},@c4237d4380] [n1,s1,r9/1:/{Table/51/1/1…-Max}]: change replicas aborted due to failed preemptive snapshot: r9: remote couldn't accept snapshot with error: [n2,s2],r9: cannot apply snapshot: snapshot intersects existing range; initiated GC: [n2,s2,r7/2:/{Table/50-Max}]
I170214 07:57:31.616228 26075 storage/replica_command.go:2397  [n1,s1,r10/1:/{Table/51/1/2…-Max},@c423b4e380] initiating a split of this range at key /Table/51/1/300 [r11]
I170214 07:57:31.621698 25074 storage/replica_raftstorage.go:414  [raftsnapshot,n1,s1,r10/1:/{Table/51/1/2…-Max},@c423b4e380] generated Raft snapshot 91e09507 at index 13
E170214 07:57:31.622415 25074 storage/queue.go:628  [raftsnapshot,n1,s1,r10/1:/{Table/51/1/2…-Max},@c423b4e380] [n1,s1,r10/1:/{Table/51/1/2…-Max}]: change replicas aborted due to failed preemptive snapshot: r10: remote couldn't accept snapshot with error: [n2,s2],r10: cannot apply snapshot: snapshot intersects existing range; initiated GC: [n2,s2,r7/2:/{Table/50-Max}]
I170214 07:57:31.627616 26108 storage/replica_command.go:2397  [n1,s1,r11/1:/{Table/51/1/3…-Max},@c4234a5c00] initiating a split of this range at key /Table/51/1/400 [r12]
I170214 07:57:31.630037 25074 storage/replica_raftstorage.go:414  [raftsnapshot,n1,s1,r11/1:/{Table/51/1/3…-Max},@c4234a5c00] generated Raft snapshot 6e13024c at index 11
E170214 07:57:31.630486 25074 storage/queue.go:628  [raftsnapshot,n1,s1,r11/1:/{Table/51/1/3…-Max},@c4234a5c00] [n1,s1,r11/1:/{Table/51/1/3…-Max}]: change replicas aborted due to failed preemptive snapshot: r11: remote couldn't accept snapshot with error: [n2,s2],r11: cannot apply snapshot: snapshot intersects existing range; initiated GC: [n2,s2,r7/2:/{Table/50-Max}]
I170214 07:57:31.638612 26149 storage/replica_command.go:2397  [n1,s1,r12/1:/{Table/51/1/4…-Max},@c422dad500] initiating a split of this range at key /Table/51/1/500 [r13]
I170214 07:57:31.640574 26180 storage/raft_transport.go:437  [n3] raft transport stream to node 2 established
I170214 07:57:31.640834 25074 storage/replica_raftstorage.go:414  [raftsnapshot,n1,s1,r12/1:/{Table/51/1/4…-Max},@c422dad500] generated Raft snapshot 2e3d4416 at index 11
I170214 07:57:31.640853 26181 storage/raft_transport.go:437  [n2] raft transport stream to node 3 established
I170214 07:57:31.641248 25704 storage/replica_raftstorage.go:414  [raftsnapshot,n3,s3,r11/3:/Table/51/1/{300-400},@c422dac000] generated Raft snapshot 8cbdb071 at index 17
E170214 07:57:31.641322 25074 storage/queue.go:628  [raftsnapshot,n1,s1,r12/1:/{Table/51/1/4…-Max},@c422dad500] [n1,s1,r12/1:/{Table/51/1/4…-Max}]: change replicas aborted due to failed preemptive snapshot: r12: remote couldn't accept snapshot with error: [n2,s2],r12: cannot apply snapshot: snapshot intersects existing range; initiated GC: [n2,s2,r7/2:/{Table/50-Max}]
E170214 07:57:31.641678 25704 storage/queue.go:628  [raftsnapshot,n3,s3,r11/3:/Table/51/1/{300-400},@c422dac000] [n3,s3,r11/3:/Table/51/1/{300-400}]: change replicas aborted due to failed preemptive snapshot: r11: remote couldn't accept snapshot with error: [n2,s2],r11: cannot apply snapshot: snapshot intersects existing range; initiated GC: [n2,s2,r7/2:/{Table/50-Max}]
I170214 07:57:31.648060 26227 storage/replica_command.go:2397  [n1,s1,r13/1:/{Table/51/1/5…-Max},@c423268380] initiating a split of this range at key /Table/51/1/600 [r14]
I170214 07:57:31.649121 25074 storage/replica_raftstorage.go:414  [raftsnapshot,n1,s1,r13/1:/{Table/51/1/5…-Max},@c423268380] generated Raft snapshot 92fa2e65 at index 10
E170214 07:57:31.649453 25074 storage/queue.go:628  [raftsnapshot,n1,s1,r13/1:/{Table/51/1/5…-Max},@c423268380] [n1,s1,r13/1:/{Table/51/1/5…-Max}]: change replicas aborted due to failed preemptive snapshot: r13: remote couldn't accept snapshot with error: [n2,s2],r13: cannot apply snapshot: snapshot intersects existing range; initiated GC: [n2,s2,r7/2:/{Table/50-Max}]
I170214 07:57:31.680643 26235 storage/replica_command.go:2397  [n1,s1,r14/1:/{Table/51/1/6…-Max},@c423176000] initiating a split of this range at key /Table/51/1/700 [r15]
I170214 07:57:31.681722 25074 storage/replica_raftstorage.go:414  [raftsnapshot,n1,s1,r14/1:/{Table/51/1/6…-Max},@c423176000] generated Raft snapshot 16a6419e at index 10
E170214 07:57:31.682243 25074 storage/queue.go:628  [raftsnapshot,n1,s1,r14/1:/{Table/51/1/6…-Max},@c423176000] [n1,s1,r14/1:/{Table/51/1/6…-Max}]: change replicas aborted due to failed preemptive snapshot: r14: remote couldn't accept snapshot with error: [n2,s2],r14: cannot apply snapshot: snapshot intersects existing range; initiated GC: [n2,s2,r7/2:/{Table/50-Max}]
I170214 07:57:31.688407 26307 storage/replica_command.go:2397  [n1,s1,r15/1:/{Table/51/1/7…-Max},@c423203c00] initiating a split of this range at key /Table/51/1/800 [r16]
I170214 07:57:31.689750 25074 storage/replica_raftstorage.go:414  [raftsnapshot,n1,s1,r15/1:/{Table/51/1/7…-Max},@c423203c00] generated Raft snapshot 0769517d at index 11
E170214 07:57:31.692114 25074 storage/queue.go:628  [raftsnapshot,n1,s1,r15/1:/{Table/51/1/7…-Max},@c423203c00] [n1,s1,r15/1:/{Table/51/1/7…-Max}]: change replicas aborted due to failed preemptive snapshot: r15: remote couldn't accept snapshot with error: [n2,s2],r15: cannot apply snapshot: snapshot intersects existing range; initiated GC: [n2,s2,r7/2:/{Table/50-Max}]
I170214 07:57:31.702704 26342 storage/replica_command.go:2397  [n1,s1,r16/1:/{Table/51/1/8…-Max},@c423263c00] initiating a split of this range at key /Table/51/1/900 [r17]
I170214 07:57:31.704694 25074 storage/replica_raftstorage.go:414  [raftsnapshot,n1,s1,r16/1:/{Table/51/1/8…-Max},@c423263c00] generated Raft snapshot 1aa36546 at index 10
E170214 07:57:31.705438 25074 storage/queue.go:628  [raftsnapshot,n1,s1,r16/1:/{Table/51/1/8…-Max},@c423263c00] [n1,s1,r16/1:/{Table/51/1/8…-Max}]: change replicas aborted due to failed preemptive snapshot: r16: remote couldn't accept snapshot with error: [n2,s2],r16: cannot apply snapshot: snapshot intersects existing range; initiated GC: [n2,s2,r7/2:/{Table/50-Max}]
I170214 07:57:32.490715 25074 storage/replica_raftstorage.go:414  [raftsnapshot,n1,s1,r8/1:/Table/51{-/1/100},@c421416700] generated Raft snapshot 05c9a62c at index 18
E170214 07:57:32.491181 25074 storage/queue.go:628  [raftsnapshot,n1,s1,r8/1:/Table/51{-/1/100},@c421416700] [n1,s1,r8/1:/Table/51{-/1/100}]: change replicas aborted due to failed preemptive snapshot: r8: remote couldn't accept snapshot with error: [n2,s2],r8: cannot apply snapshot: snapshot intersects existing range [n2,s2,r7/2:/{Table/50-Max}]
I170214 07:57:32.491399 25074 storage/replica_raftstorage.go:414  [raftsnapshot,n1,s1,r7/1:/Table/5{0-1},@c420bfca80] generated Raft snapshot 76a9a6a6 at index 17
I170214 07:57:32.492402 25074 storage/store.go:3278  [raftsnapshot,n1,s1,r7/1:/Table/5{0-1},@c420bfca80] streamed snapshot: kv pairs: 10, log entries: 7, 1ms
I170214 07:57:32.492618 26393 storage/replica_raftstorage.go:596  [n2,s2,r7/2:/{Table/50-Max},@c421a3aa80] applying Raft snapshot at index 17 (id=76a9a6a6, encoded size=149161, 1 rocksdb batches, 7 log entries)
I170214 07:57:32.493233 26393 storage/replica_raftstorage.go:604  [n2,s2,r7/2:/Table/5{0-1},@c421a3aa80] applied Raft snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170214 07:57:32.590767 25704 storage/replica_raftstorage.go:414  [raftsnapshot,n3,s3,r11/3:/Table/51/1/{300-400},@c422dac000] generated Raft snapshot 1cecae36 at index 19
I170214 07:57:32.591395 25704 storage/store.go:3278  [raftsnapshot,n3,s3,r11/3:/Table/51/1/{300-400},@c422dac000] streamed snapshot: kv pairs: 110, log entries: 9, 0ms
I170214 07:57:32.591486 26420 storage/replica_raftstorage.go:596  [n2,s2,r11/2:{-},@c4237d5180] applying Raft snapshot at index 19 (id=1cecae36, encoded size=16008, 1 rocksdb batches, 9 log entries)
I170214 07:57:32.591761 26420 storage/replica_raftstorage.go:604  [n2,s2,r11/2:/Table/51/1/{300-400},@c4237d5180] applied Raft snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170214 07:57:32.691000 25074 storage/replica_raftstorage.go:414  [raftsnapshot,n1,s1,r12/1:/Table/51/1/{400-500},@c422dad500] generated Raft snapshot 67e925b1 at index 16
I170214 07:57:32.691989 25074 storage/store.go:3278  [raftsnapshot,n1,s1,r12/1:/Table/51/1/{400-500},@c422dad500] streamed snapshot: kv pairs: 110, log entries: 6, 1ms
I170214 07:57:32.692115 26398 storage/replica_raftstorage.go:596  [n2,s2,r12/2:{-},@c421ff6380] applying Raft snapshot at index 16 (id=67e925b1, encoded size=15512, 1 rocksdb batches, 6 log entries)
I170214 07:57:32.692379 26398 storage/replica_raftstorage.go:604  [n2,s2,r12/2:/Table/51/1/{400-500},@c421ff6380] applied Raft snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170214 07:57:32.694098 25074 storage/replica_raftstorage.go:414  [raftsnapshot,n1,s1,r16/1:/{Table/51/1/8…-Max},@c423263c00] generated Raft snapshot e23dfc33 at index 12
I170214 07:57:32.694649 25074 storage/store.go:3278  [raftsnapshot,n1,s1,r16/1:/{Table/51/1/8…-Max},@c423263c00] streamed snapshot: kv pairs: 212, log entries: 2, 0ms
I170214 07:57:32.694845 26350 storage/replica_raftstorage.go:596  [n2,s2,r16/2:{-},@c42193c000] applying Raft snapshot at index 12 (id=e23dfc33, encoded size=26925, 1 rocksdb batches, 2 log entries)
I170214 07:57:32.695131 26350 storage/replica_raftstorage.go:604  [n2,s2,r16/2:/{Table/51/1/8…-Max},@c42193c000] applied Raft snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170214 07:57:32.696235 25074 storage/replica_raftstorage.go:414  [raftsnapshot,n1,s1,r9/1:/Table/51/1/{100-200},@c4237d4380] generated Raft snapshot f39b5412 at index 16
I170214 07:57:32.697513 25074 storage/store.go:3278  [raftsnapshot,n1,s1,r9/1:/Table/51/1/{100-200},@c4237d4380] streamed snapshot: kv pairs: 110, log entries: 6, 1ms
I170214 07:57:32.697670 26379 storage/replica_raftstorage.go:596  [n2,s2,r9/2:{-},@c423acea80] applying Raft snapshot at index 16 (id=f39b5412, encoded size=15341, 1 rocksdb batches, 6 log entries)
I170214 07:57:32.697990 26379 storage/replica_raftstorage.go:604  [n2,s2,r9/2:/Table/51/1/{100-200},@c423acea80] applied Raft snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170214 07:57:32.704145 25074 storage/replica_raftstorage.go:414  [raftsnapshot,n1,s1,r10/1:/Table/51/1/{200-300},@c423b4e380] generated Raft snapshot b0532c18 at index 16
I170214 07:57:32.704638 25074 storage/store.go:3278  [raftsnapshot,n1,s1,r10/1:/Table/51/1/{200-300},@c423b4e380] streamed snapshot: kv pairs: 110, log entries: 6, 0ms
I170214 07:57:32.704782 26455 storage/replica_raftstorage.go:596  [n2,s2,r10/2:{-},@c4237d4e00] applying Raft snapshot at index 16 (id=b0532c18, encoded size=15430, 1 rocksdb batches, 6 log entries)
I170214 07:57:32.705036 26455 storage/replica_raftstorage.go:604  [n2,s2,r10/2:/Table/51/1/{200-300},@c4237d4e00] applied Raft snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170214 07:57:32.706151 25074 storage/replica_raftstorage.go:414  [raftsnapshot,n1,s1,r15/1:/Table/51/1/{700-800},@c423203c00] generated Raft snapshot fe31d077 at index 16
I170214 07:57:32.706731 25074 storage/store.go:3278  [raftsnapshot,n1,s1,r15/1:/Table/51/1/{700-800},@c423203c00] streamed snapshot: kv pairs: 110, log entries: 6, 0ms
I170214 07:57:32.706885 26365 storage/replica_raftstorage.go:596  [n2,s2,r15/2:{-},@c423221c00] applying Raft snapshot at index 16 (id=fe31d077, encoded size=15508, 1 rocksdb batches, 6 log entries)
I170214 07:57:32.707152 26365 storage/replica_raftstorage.go:604  [n2,s2,r15/2:/Table/51/1/{700-800},@c423221c00] applied Raft snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170214 07:57:32.708108 25074 storage/replica_raftstorage.go:414  [raftsnapshot,n1,s1,r14/1:/Table/51/1/{600-700},@c423176000] generated Raft snapshot b541ad41 at index 16
I170214 07:57:32.709045 25074 storage/store.go:3278  [raftsnapshot,n1,s1,r14/1:/Table/51/1/{600-700},@c423176000] streamed snapshot: kv pairs: 110, log entries: 6, 0ms
I170214 07:57:32.709193 26461 storage/replica_raftstorage.go:596  [n2,s2,r14/2:{-},@c423263500] applying Raft snapshot at index 16 (id=b541ad41, encoded size=15508, 1 rocksdb batches, 6 log entries)
I170214 07:57:32.709446 26461 storage/replica_raftstorage.go:604  [n2,s2,r14/2:/Table/51/1/{600-700},@c423263500] applied Raft snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170214 07:57:32.711269 25074 storage/replica_raftstorage.go:414  [raftsnapshot,n1,s1,r13/1:/Table/51/1/{500-600},@c423268380] generated Raft snapshot 7707075c at index 16
I170214 07:57:32.712849 25074 storage/store.go:3278  [raftsnapshot,n1,s1,r13/1:/Table/51/1/{500-600},@c423268380] streamed snapshot: kv pairs: 110, log entries: 6, 1ms
I170214 07:57:32.712999 26401 storage/replica_raftstorage.go:596  [n2,s2,r13/2:{-},@c423202380] applying Raft snapshot at index 16 (id=7707075c, encoded size=15508, 1 rocksdb batches, 6 log entries)
I170214 07:57:32.713242 26401 storage/replica_raftstorage.go:604  [n2,s2,r13/2:/Table/51/1/{500-600},@c423202380] applied Raft snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170214 07:57:33.394777 25074 storage/replica_raftstorage.go:414  [raftsnapshot,n1,s1,r8/1:/Table/51{-/1/100},@c421416700] generated Raft snapshot b16f9504 at index 21
I170214 07:57:33.400965 25074 storage/store.go:3278  [raftsnapshot,n1,s1,r8/1:/Table/51{-/1/100},@c421416700] streamed snapshot: kv pairs: 211, log entries: 11, 3ms
I170214 07:57:33.402346 26549 storage/replica_raftstorage.go:596  [n2,s2,r8/2:{-},@c423ace000] applying Raft snapshot at index 21 (id=b16f9504, encoded size=311084, 1 rocksdb batches, 11 log entries)
I170214 07:57:33.406452 26549 storage/replica_raftstorage.go:604  [n2,s2,r8/2:/Table/51{-/1/100},@c423ace000] applied Raft snapshot in 4ms [clear=0ms batch=0ms entries=1ms commit=3ms]
W170214 07:57:35.741685 26563 storage/replica.go:1679  [n1,s1,r14/1:/Table/51/1/{600-700},@c423176000] context deadline exceeded while in command queue: ResolveIntentRange [/Table/51/1/600,/Table/51/1/700)
W170214 07:57:35.741858 26562 storage/replica.go:1679  [n1,s1,r17/1:/{Table/51/1/9…-Max},@c421e75500] context deadline exceeded while in command queue: ResolveIntentRange [/Table/51/1/900,/Table/52)
W170214 07:57:35.741971 26489 storage/intent_resolver.go:352  [n1,s1,r8/1:/Table/51{-/1/100}]: failed to resolve intents: context deadline exceeded
I170214 07:57:57.498134 25021 gossip/gossip.go:1192  [n1] node has connected to cluster via gossip
I170214 07:57:57.498277 25021 storage/stores.go:312  [n1] wrote 2 node addresses to persistent storage
I170214 07:58:00.938720 25280 gossip/gossip.go:1192  [n2] node has connected to cluster via gossip
I170214 07:58:00.938844 25280 storage/stores.go:312  [n2] wrote 2 node addresses to persistent storage
I170214 07:58:05.289901 25428 gossip/gossip.go:1192  [n3] node has connected to cluster via gossip
I170214 07:58:05.290126 25428 storage/stores.go:312  [n3] wrote 2 node addresses to persistent storage


ERROR: signal: killed

Makefile:173: recipe for target 'stress' failed
make: *** [stress] Error 1
```

--- a/pkg/ccl/sqlccl/backup_test.go
+++ b/pkg/ccl/sqlccl/backup_test.go
@@ -392,7 +392,6 @@ func TestBackupRestoreBank(t *testing.T) {
 
 func TestBackupAsOfSystemTime(t *testing.T) {
 	defer leaktest.AfterTest(t)()
-	t.Skip("#13575")
 	if !storage.ProposerEvaluatedKVEnabled() {
 		t.Skip("command WriteBatch is not allowed without proposer evaluated KV")
 	}

--- a/pkg/ccl/storageccl/export.go
+++ b/pkg/ccl/storageccl/export.go
@@ -91,29 +91,23 @@ func evalExport(
 	// TODO(dan): Move all this iteration into cpp to avoid the cgo calls.
 	// TODO(dan): Consider checking ctx periodically during the MVCCIterate call.
 	var entries int64
-	for {
-		entries = 0
-		iter := engineccl.NewMVCCIncrementalIterator(batch)
-		defer iter.Close()
-		iter.Reset(args.Key, args.EndKey, args.StartTime, h.Timestamp)
-		for ; iter.Valid(); iter.Next() {
-			key, value := iter.Key(), iter.Value()
-			if log.V(3) {
-				log.Infof(ctx, "Export %+v %+v", key, value)
-			}
-			entries++
-			if err := sst.Add(engine.MVCCKeyValue{Key: key, Value: value}); err != nil {
-				return storage.EvalResult{}, errors.Wrapf(err, "adding key %s", key)
-			}
+	iter := engineccl.NewMVCCIncrementalIterator(batch)
+	defer iter.Close()
+	iter.Reset(args.Key, args.EndKey, args.StartTime, h.Timestamp)
+	for ; iter.Valid(); iter.Next() {
+		key, value := iter.Key(), iter.Value()
+		if log.V(3) {
+			log.Infof(ctx, "Export %+v %+v", key, value)
 		}
-		err := iter.Error()
-		if _, ok := err.(*roachpb.WriteIntentError); ok {
-			continue
+		entries++
+		if err := sst.Add(engine.MVCCKeyValue{Key: key, Value: value}); err != nil {
+			return storage.EvalResult{}, errors.Wrapf(err, "adding key %s", key)
 		}
-		if err != nil {
-			return storage.EvalResult{}, err
-		}
-		break
+	}
+	if err := iter.Error(); err != nil {
+		// The error may be a WriteIntentError. In which case, returning it will
+		// cause this command to be retried.
+		return storage.EvalResult{}, err
 	}
 
 	if entries == 0 {

